src/ontology/uberon-edit.obo

```diff
@@ -219794,6 +219794,7 @@ synonym: "stomodeal nervous system" RELATED [FlyBase:FBrf0111704]
 is_a: UBERON:0011216 ! organ system subdivision
 relationship: in_taxon NCBITaxon:6656 ! Arthropoda
 relationship: part_of UBERON:0001017 ! central nervous system
+relationship: is_a UBERON:8910000 ! stomatogastric nervous system
```

Comment:
```suggestion
is_a: UBERON:8910000 ! stomatogastric nervous system
```
The syntax for this SubClassOf  assertion still does not look correct. You can see what I mean If you open `uberon-edit.obo` in Protege and check if the "insect stomatogastric nervous system" is a sub-class of "stomatogastric nervous system".